pkg/cmd/cli/restore/create.go

```diff
@@ -279,6 +283,13 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		},
 	}
 
+	if len([]string(o.StatusIncludeResources)) > 0 {
```

Comment:
Should o.StatusExcludeResources length be checked, too?

Reply:
Well, I believe if you don't include any resources, it won't trigger anything anyway